js/interactive-guides/step-content.js

```diff
@@ -153,20 +153,37 @@ var stepContent = (function() {
       return;
     }
     var description = step.description;    
-    var jointDescription = description;
     if ($.isArray(description)) {
-      jointDescription = description.join("<br/>");
+        $.each(description, function(i, desc) {
+            if (desc) {
+                //TODO: check for header or <li> tag and ignore these
+                if (__containsHTMLTag(desc)) {
```

Comment:
This can be updated to if(!_containsHTMLTag(desc)){ // code } instead of an empty tag.

Reply:
Good catch. I forgot to optimize the logic.